setup.py

```diff
@@ -213,6 +213,10 @@ def read(*names, **kwargs):
             # used by yg.lockfile
             'contextlib2', 'pytz', 'tempora', 'jaraco.functools',
         'zc.lockfile >= 2.0.0, < 3.0.0',
+
+        #for packaging and uploading
+        'pip-tools >=5.3.0, < 6.0.0',
+        'github-release-retry >=1.0.6, < 2.0.0; python_version > "3"'
```

Comment:
These two are dev-only... they should not be in setup.py ... instead put them either:
- in a dedicated requirements files in etc/scripts with a README.rst that explains how to use that... 
- OR in `etc/conf/dev/requirements_base.txt`

Reply:
Actually, `in etc/conf/dev/requirements_base.txt` cannot work as it would require for now that the wheels be in /thirdparty until we change and update the configure.py.... So please use a dedicated requirements files in etc/scripts with a README.rst that explains how to use that...